test/run_tests.py

```diff
@@ -10,26 +10,43 @@
 
 Usage:
 
-  run_tests.py interpreter browser [options]
+  run_tests.py browser [options]
 
 Arguments:
 
-  interpreter:  Any Python interpreter supported by the library (e.g. `python`,
-                `jython`, `c:\\Python27\\python.exe`)
-  browser:      Any browser supported by the library (e.g. `chrome`, `firefox`)
-  options:      Additional command line options passed to Robot Framework
+  browser:        Any browser supported by the library (e.g. `chrome`or
+                  `firefox`)
+  --interpreter:  Any Python interpreter supported by the library (e.g.
+                  `python`, `jython` or `c:\\Python27\\python.exe`). By
+                  default set to `python`.
+  --suite:        Selects the test suites by name.
+  --scusername:   Username to access Sauce Labs to order browsers when
+                  running test from local computer.
+  --sckey:        Access key for Sauce Labs account.
```

Comment:
Yes it does and it's actually quite good help system. Dividing this with readme file and argparse is good idea. At least the argparse help must be enhanced.